blocks/text/text.css

```diff
@@ -36,24 +36,20 @@ main .block.text.black p {
 }
 
 main .block.text.highlight {
-  margin-top: 5rem;
+  margin-top: 1.5rem;
+  margin-bottom: 5rem;
+  padding: 1.875rem 2.5rem;
+  background: var(--transparent-blue-light-color);
+  border-left: 5px solid var(--health-beauty-orange);
 }
 
-
-main .block.text.highlight h2 {
-  margin-bottom: .5rem;
-  text-align: center;
+main .block.text.highlight .text-title {
+  color: var(--health-beauty-orange);
+  margin-bottom: unset;
 }
 
-main .block.text.highlight a.button.primary {
-  color: var(--secondary);
-  background: none;
-  font-weight: var(--font-weight-medium);
-  font-size: var(--body-font-size-m);
-  line-height: 1.5em;
-  text-decoration: none;
-  padding: 0;
-  height: auto;
+.block.quote.highlight .text-content {
```

Comment:
Why this change ?

Reply:
I am talking about, i think this is not needed
`main .block.quote.highlight .text-content`

Reply:
@jindaliiita I just keep the same with the original site. In EDS the default one is `#3c4350`, there is no big difference between the two in eyes, I removed it.

Original site:
<img width="483" alt="image" src="https://github.com/hlxsites/sunstar-foundation/assets/57093988/f06a38ad-4a72-4ac9-8bf2-f9161b468a73">